src/Command.php

```diff
@@ -126,6 +126,9 @@ private static function cleanPath(string $path, bool $rtrim = false): string
             $path = \rtrim($path, '/');
         }
 
+        $path = \trim($path, '"');
```

Comment:
There is a flag argument to check whether to trim the value or not, reasons where inconsistent path checks. Adding `trim()` here makes the arg useless, while it is useful not to trim sometimes.